client/src/includes/tabs.js

```diff
@@ -230,6 +230,13 @@ class Tabs {
         }
       }
     });
+
+    // Dispatch tab-changed on window load
+    window.onload = () => {
```

Comment:
```suggestion
    document.addEventListener('DOMContentLoaded', () => {
```

We should be using `addEventListener` for this rather than the global callback. Technically all of the tabs initialisations are meant to be happening after `DOMContentLoaded` anyway, so I don’t see why we’d even want to dispatch the event only on `load`.